### DISCUSSION TITLE
I am lost

### DISCUSSION THREAD

**Discussion by andremartin1979** [GyulyVGC/sniffnet#112] [Q&A] [answered]
Installed sniffnet all that was installed was a bin folder with sniffnet.exe inside.  Tried running it got error came back saw I needed dependencies downloaded them.  I am having trouble with the LIB environment and placing the sdk's LIB there because I have no idea how find that environment let alone make a folder for it and then how do I link it.  I apologize for getting into something I have no clue about I am trying to learn.  I appreciate any help and I thank you all for taking the time to help.

**Comment by GyulyVGC** [comment]
You can follow this simple guide which will show you how to create an environment variable on Windows: https://www.howtogeek.com/787217/how-to-edit-environment-variables-on-windows-10-or-11/

The name of the variable you have to add is LIB, while the value is the full path of the SDK's LIB/x64 folder.

Let me know in case of further doubts.

**Reply by andremartin1979** [reply]
Will this procedure work with window 8.1?

**Reply by GyulyVGC** [reply]
Yes, absolutely

**Reply by andremartin1979** [reply]
Thanks, I will follow up when I get it done.

**Reply by andremartin1979** [reply]
Alright I got it done, started sniffnet and it actually opened up.  Awesome thank you so much for your help

**Reply by GyulyVGC** [reply, marked as answer]
That's awesome to hear! 
Don't forget to mark my comment as answer so that other people will be able to see this conversation as solved. 
Have a good day!

**Reply by andremartin1979** [reply]
Sorry I didn't even notice the reply back.  I have some family stuff.

**Reply by GyulyVGC** [reply]
No problem!

**Reply by andremartin1979** [reply]
Yeah and right getting everything set up and ready a couple of days later my computer had enough of my constant need to learn and died right as I was in the middle of downloading a nice bit of data, if you know what I mean.  
 
She's no 10 and for sure isn't some prettied-up and patched freakish 11.  I mean where why not 12 hell 13 while they're at it?  Who cares we all know nothing is ever completely done deadlines are well deadlines.  How do we expect these billionaires to have all these charities if we don't make some people go bankrupt and live in the streets?  Lol oh my goodness I needed that.  If you think what I just wrote is a little distasteful just know it was way worse.
